dev-libs/uriparser/uriparser-0.9.1~20190116.recipe

```diff
@@ -42,25 +49,53 @@ REQUIRES_devel="
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
 	"
+if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
+BUILD_REQUIRES="$BUILD_REQUIRES
+	devel:libgtest$secondaryArchSuffix
+	"
+fi
+
 BUILD_PREREQUIRES="
-	cmd:aclocal
-	cmd:autoconf
-	cmd:automake
-	cmd:autoreconf
+	cmd:cmake
 	cmd:gcc$secondaryArchSuffix
 	cmd:libtoolize$secondaryArchSuffix
 	cmd:make
 	"
+if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
+BUILD_PREREQUIRES="$BUILD_PREREQUIRES
+	cmd:dot
+	cmd:doxygen
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:qhelpgenerator$secondaryArchSuffix
+	"
+fi
+
 
 defineDebugInfoPackage uriparser$secondaryArchSuffix \
 	"$libDir"/liburiparser.so.$libVersion
 
+if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
 BUILD()
 {
-	autoreconf
-	runConfigure ./configure --disable-test --disable-doc
+	cmake -DCMAKE_INSTALL_PREFIX=$prefix \
+		-DCMAKE_INSTALL_BINDIR:PATH=$binDir \
+		-DCMAKE_INSTALL_INCLUDEDIR:PATH=$includeDir \
+		-DCMAKE_INSTALL_LIBDIR:PATH=$libDir \
+		-DCMAKE_INSTALL_DOCDIR:PATH=$docDir
```

Comment:
Try using `$cmakeDirArgs` instead of all this, that's what it's for.

Reply:
I'll be sure to try it, not a big fan for cmake and seen it passing by, but never had much luck in using it :)

Reply:
Working, thanks! :)